src/liquid/auth/authentication_form.html

```diff
@@ -1,39 +1,48 @@
 <form id="form_authenticate" hx-post="/authenticate" hx-target="this" hx-swap="outerHTML">
-  <!-- want to use tiles, they are super weird with forms in them -->
-  <div class="box" style="margin: 20px; padding: 50px;">
-    {% if errors %}
-      <span class='tag is-warning is-overlay'>Login Failed. Please Try Again</span>
-    {% endif %}
-
-    <div class="field">
-      <label class="label">Instance</label>
-      <div class="control">
-        <div class="select is-fullwidth">
-          <select name="domain">
-              {% for domain in domains %}
-                <option value="{{ domain.domain }}">{{ domain.friendly }}</option>
-              {% endfor %}
-          </select>
-        </div>
-      </div>
-    </div>
-    <div class="field">
-      <label class="label">Credentials</label>
-      <div class="control">
-          <input name="username" class="input" type="text" placeholder="username">
-      </div>
+  <div>
+    <label>Instance</label>
+    <div class="custom-select">
+      <select class="dropdown-input" name="domain">
+        {% for domain in domains %}
+        <option value="{{ domain.domain }}">{{ domain.friendly }}</option>
+        {% endfor %}
+      </select>
     </div>
-    <div class="field">
-        <div class="control">
-            <input name="password" class="input" type="password" placeholder="password">
-        </div>
-    </div>
-    <div class="field is-grouped is-grouped-right">
-      <p class="control">
-        <button class="button is-primary">
-          Sign In
-        </button>
-      </p>
+  </div>
+
+  <div>
+    <label>Credentials</label>
+    <div>
+      <div><input name="username" type="text" placeholder="Username" required></div>
+      <div><input name="password" type="password" placeholder="Password" required></div>
     </div>
   </div>
+
+  <div id="error-msg" class="error center">
+    <span>Login Failed. Please Try Again</span>
+  </div>
+
+  <div id="spinner" class="spinner"></div>
```

Comment:
Can we have this spinner show in the login button? I feel like that would be a better indicator of the operation's progress

Reply:
Take a look at https://v1.htmx.org/attributes/hx-indicator/ and how it's currently used eg in https://github.com/medic/cht-user-management/blob/main/src/liquid/place/bulk_create_form.html